cmd/kube-apiserver/app/server.go

```diff
@@ -127,6 +127,9 @@ func Run(s *options.ServerRunOptions) error {
 		return err
 	}
 
+	// Use protobufs for self-communication.
+	genericConfig.LoopbackClientConfig.ContentConfig.ContentType = "application/vnd.kubernetes.protobuf"
```

Comment:
@liggitt - do we want this to be set by default for every apiserver?
I though not - and in that case we need to overwrite it in kube-apiserver.